extensions/cornerstone/src/init.js

```diff
@@ -4,30 +4,32 @@ import csTools from 'cornerstone-tools';
 import initCornerstoneTools from './initCornerstoneTools.js';
 import queryString from 'query-string';
 
-function fallbackMetaDataProvider (type, imageId) {
+function fallbackMetaDataProvider(type, imageId) {
   if (!imageId.includes('wado?requestType=WADO')) {
-    return
+    return;
   }
 
-  // If you call for an WADO-URI imageId and get no 
+  // If you call for an WADO-URI imageId and get no
   // metadata, try reformatting to WADO-RS imageId
   const qs = queryString.parse(imageId);
-  const wadoRoot = window.store.getState().servers.servers[0].wadoRoot
-  const wadoRsImageId = `wadors:${wadoRoot}/studies/${qs.studyUID}/series/${qs.seriesUID}/instances/${qs.objectUID}/frames/${qs.frame || 1}`;
+  const wadoRoot = window.store.getState().servers.servers[0].wadoRoot;
+  const wadoRsImageId = `wadors:${wadoRoot}/studies/${qs.studyUID}/series/${
+    qs.seriesUID
+  }/instances/${qs.objectUID}/frames/${qs.frame || 1}`;
 
   return cornerstone.metaData.get(type, wadoRsImageId);
 }
 
 // Add this fallback provider with a low priority so it is handled last
 cornerstone.metaData.addProvider(fallbackMetaDataProvider, -1);
 
-
 /**
  *
  * @param {object} configuration
  * @param {Object|Array} configuration.csToolsConfig
  */
-export default function init(configuration = {}) {
+export default function init(services, configuration = {}) {
+  const { UINotificationService } = services;
```

Comment:
We should probably do: `UiNotificationService`

With `UINotificationService`, it makes it more ambigous when we should break from pascal casing. We had an issue with this earlier for `SeriesInstanceUid` (should it be `UID`, `UId`, or `Uid`?)

Reply:
UINotificationService => User Interface Notification Service is more natural to me because im actually contracting each word in my mind and remembering that people usually write UI/UX but we can definitely change this to use a different casing.

Reply:
@igoroctaviano, I'm generally cool with whatever (with regards to naming), so long as we are consistent 👍